assets/xml/objects/object_yb.xml

```diff
@@ -1,57 +1,58 @@
-        <Limb Name="object_yb_Standardlimb_005EA0" Type="Standard" Offset="0x5EA0" />
-        <Limb Name="object_yb_Standardlimb_005EAC" Type="Standard" Offset="0x5EAC" />
-        <Limb Name="object_yb_Standardlimb_005EB8" Type="Standard" Offset="0x5EB8" />
-        <Limb Name="object_yb_Standardlimb_005EC4" Type="Standard" Offset="0x5EC4" />
-        <Limb Name="object_yb_Standardlimb_005ED0" Type="Standard" Offset="0x5ED0" />
-        <Limb Name="object_yb_Standardlimb_005EDC" Type="Standard" Offset="0x5EDC" />
-        <Limb Name="object_yb_Standardlimb_005EE8" Type="Standard" Offset="0x5EE8" />
-        <Skeleton Name="object_yb_Skel_005F48" Type="Flex" LimbType="Standard" Offset="0x5F48" />
+        <DList Name="gYbPantsAndBellyButtonDL" Offset="0x3400" />
+        <DList Name="gYbRightThighDL" Offset="0x3610" />
+        <DList Name="gYbRightCalfDL" Offset="0x37B0" />
+        <DList Name="gYbRightFoodDL" Offset="0x3900" />
+        <DList Name="gYbLeftThighDL" Offset="0x3BC8" />
+        <DList Name="gYbLeftCalfDL" Offset="0x3D68" />
+        <DList Name="gYbLeftFootDL" Offset="0x3EB8" />
+        <DList Name="gYbEndDisplayList1DL" Offset="0x4108" />
+        <DList Name="gYbUpperTorsoDL" Offset="0x4110" />
+        <DList Name="gYbHeadDL" Offset="0x4310" />
+        <DList Name="gYbPonyTailDL" Offset="0x47F8" />
+        <DList Name="gYbEndDisplayList2DL" Offset="0x4970" />
+        <DList Name="gYbRightUpperArmDL" Offset="0x4978" />
+        <DList Name="gYbRightForArmDL" Offset="0x4AA0" />
+        <DList Name="gYbRightHandDL" Offset="0x4BC0" />
+        <DList Name="gYbLeftUpperArmDL" Offset="0x4D98" />
+        <DList Name="gYbLeftForArmDL" Offset="0x4EC0" />
+        <DList Name="gYbLeftHandDL" Offset="0x4FE0" />
+        <Texture Name="gYbTLUT" OutName="yb_tlut" Format="rgba16" Width="16" Height="16" Offset="0x51B8" />
+        <Texture Name="gYbForheadBlotchTex" OutName="yb_forhead_blotch" Format="ci8" Width="8" Height="8" Offset="0x53B8" />
+        <Texture Name="gYbPonyTailTex" OutName="yb_ponytail" Format="ci8" Width="8" Height="16" Offset="0x53F8" />
+        <Texture Name="gYbFingersTex" OutName="yb_fingers" Format="ci8" Width="16" Height="16" Offset="0x5478" />
+        <Texture Name="gYbToesTex" OutName="yb_toes" Format="ci8" Width="16" Height="16" Offset="0x5578" />
+        <Texture Name="gYbHairLineTex" OutName="yb_hairline" Format="ci8" Width="16" Height="16" Offset="0x5678" />
+        <Texture Name="gYbBellybuttonTex" OutName="yb_bellybutton" Format="ci8" Width="8" Height="16" Offset="0x5778" />
+        <Texture Name="gYbNoseTex" OutName="yb_nose" Format="ci8" Width="16" Height="16" Offset="0x57F8" />
+        <Texture Name="gYbEyeTex" OutName="yb_eye" Format="ci8" Width="32" Height="32" Offset="0x58F8" />
+        <Texture Name="gYbPantsTex" OutName="yb_pants" Format="i8" Width="16" Height="16" Offset="0x5CF8" />
+        <Limb Name="gYbRoot1Limb" Type="Standard" Offset="0x5DF8" />
```

Comment:
Instead of root#Limb, could you perhaps describe what it is the root of? If it is root of everything just RootLimb would be fine.

Reply:
I don't know what it is, it wont highlight in z64Utils. I was told to name it but this is guess work